Add metadata (#30138)
grafana/grafana: docs/sources/enterprise/license/activate-license.md

To download your Grafana Enterprise license:

1. Sign in to your [Grafana Cloud](https://grafana.com) account.
1. Go to **My Account** and select an organization from the drop-down menu at the top left of the page. On the Overview page for each organization, you can see a section for Grafana Enterprise licenses. Click **Details** next to a license.

> 1. At the bottom of the license details page, select **Download token** to download the license.jwt file that contains your license.

Reviewer: ```suggestion
1. At the bottom of the license details page, select **Download token** to download the `license.jwt` file that contains your license.
```

1. Go to **My Account** and select an organization from the drop-down menu at the top left of the page. On the Overview page for each organization, you can see a section for Grafana Enterprise licenses. Click **Details** next to a license.
1. At the bottom of the license details page, select **Download token** to download the license.jwt file that contains your license.

## Step 2. Add your license to a Grafana instance

> There are three different ways to add the license to a Grafana instance:

Reviewer: Let's generalize this and remove the baked in number:
```suggestion
There is more than one way to add the license to a Grafana instance:
```

Reviewer 2: Added manually. (Had messed up the first merge conflict so couldn't apply the suggestion automatically)

> ### Upload the license file through the Grafana Server Administrator page

Reviewer: ```suggestion
### Upload the license file via the Grafana server administrator page
```

Reviewer 2: Added manually.

This is the preferred option for single instance installations of Grafana Enterprise.

1. Sign in as a Grafana server admin.
1. Navigate to **Server Admin > Upgrade** within Grafana.
1. Click **Upload license token file**.

> 1. Select your license file, and upload it.

Reviewer: ```suggestion
1. Sign in as a Grafana server administrator.
1. Navigate to **Server Admin > Upgrade** within Grafana.
1. Click **Upload license token file**.
1. Select your license file, and upload it.
```

Reviewer 2: Added manually.

> ### Place the license.jwt file in Grafana's data folder

Reviewer: ```suggestion
### Put the `license.jwt` file into the data directory of Grafana
```

> The data folder is usually `/var/lib/grafana` on Linux systems.

Reviewer: ```suggestion
On Linux systems, the data directory is usually at `/var/lib/grafana`.
```

Reviewer 2: Added manually
